workflow/apm/services.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Services
+sidebarTitle: Services
+---
+
+The Services page displays all of your APM data in a unified view. A service groups together interconnected endpoints, queries, or jobs within your application. This is useful for contextualizing broader issues within your software infrastructure. 
+
+This landing page displays common metrics like requests, latency, and errors. Use the __Resources__, __Traces__, and __Logs__ tabs to drill into your data. This page is available across other APM and log pages to easily understand how data is related to other aspects of your application. Clicking on any of the charts or tables within this page will navigate you to a deeper view of that selected service, including resources, traces, and metrics.
```

Comment:
Think we can simplify this a little bit. "The _Resources_, _Traces_, and _Logs_ tabs provide specific contextual views of your service. Clicking on any of the charts or tables..."

The middle sentence of "This page is available across other APM and log pages..." is confusing, becuase technically the Service page we're talking about is only available from the Service navigation.